website/src/components/TelemetryLink.tsx

```diff
@@ -0,0 +1,29 @@
+import Link from '@docusaurus/Link';
+
+type TelemetryLinkProps = {
+  to: string;
```

Comment:
please use `href` as it is a _standard_

Reply:
AFAIK `to` is the standard there https://docusaurus.io/docs/docusaurus-core#to-string